docs/documentation/generate.md

```diff
@@ -5,20 +5,24 @@
 ## Overview
 `ng generate [name]` generates the specified schematic
 
+## Alias
+g - `ng g [name]`
+
 ## Available Schematics:
- - [class](generate/class)
- - [component](generate/component)
- - [directive](generate/directive)
- - [enum](generate/enum)
- - [guard](generate/guard)
- - [interface](generate/interface)
- - [module](generate/module)
- - [pipe](generate/pipe)
- - [service](generate/service)
+ - [class](generate/class.md)
```

Comment:
Please see the comment at the top of this file.  The `.md` extension should not be present.

Reply:
I can remove them but that means the links dont work in git hub.

Reply:
Yes, that is correct.  The files are published to the wiki on release.  They are not intended to be used directly from the repository.

Please squash and change the commit message type to `docs`.

Thank you.

Reply:
Im sorry what do you mean by Please squash and change the commit message type to docs

Reply:
There should only be one commit with a message starting with `docs(@angular/cli):`.
If you prefer, the caretaker can handle it when they merge.

However, for information on rebasing please see this guide: https://help.github.com/articles/about-git-rebase/

Reply:
Done. I am familiar with rebasing just hadn't done a squash before so didn't know that was an option. Thanks for the info.